### PR TITLE
WIP: Webhook wechat

### PR DESCRIPTION
**Pull request by 6543**
use #5252 as source ... rebase & apply patches & refacotr ...

credits to: @rocklau

## PS: everybody is welcome to pick this PR up ...

### CODE REVIEW

**Comment by 6543** [comment]
@rocklau may you have a look at?  (can't test this [dont have a wechat jet])

File: templates/repo/settings/webhook/list.tmpl [hunk 1]
```diff
@@ -26,6 +26,9 @@
 				<a class="item" href="{{.BaseLink}}/msteams/new">
 					<img class="img-10" src="{{StaticUrlPrefix}}/img/msteams.png">Microsoft Teams
 				</a>
+				<a class="item" href="{{.BaseLink}}/settings/hooks/workwechat/new">
```
Comment (6543) [inline templates/repo/settings/webhook/list.tmpl]:
```suggestion
				<a class="item" href="{{.BaseLink}}/workwechat/new">
```

**Comment by rocklau** [comment]
Recommend integrate with this package.
https://github.com/xen0n/go-workwx

**Comment by 6543** [comment]
no stalebot no ;)

**Review by Zoranner — CHANGES_REQUESTED** [review]
LGTM

File: templates/repo/settings/webhook/list.tmpl [hunk 2]
```diff
@@ -30,7 +30,11 @@
 					<img class="img-10" src="{{StaticUrlPrefix}}/img/feishu.png">Feishu
 				</a>
 				<a class="item" href="{{.BaseLink}}/matrix/new">
-                	<img class="img-10" src="{{StaticUrlPrefix}}/img/matrix.svg">Matrix
+					<img class="img-10" src="{{StaticUrlPrefix}}/img/matrix.svg">Matrix
+
```
Comment (6543) [inline templates/repo/settings/webhook/list.tmpl]:
```suggestion
```

**Review by Zoranner — CHANGES_REQUESTED** [review]
LGTM

**Comment by lunny** [comment]
Since #12310 merge, this should be updated.

**Comment by 6543** [comment]
will do ...

**Comment by delvh** [comment]
Is this not closed by #15910?

**Comment by 6543** [comment]
Closed by  #15910

**Pull request closed without merging**